### DISCUSSION TITLE
What is the "Positive Rate Dash"/"DDC Positive Rate" and why is it different from the other Positive Rate?

### DISCUSSION THREAD

**Discussion by liberatorfey** [djay/covidthailand#216] [Q&A] [answered]
What are mean column Positive Rate Dash in file cases_by_province.csv

**Comment by djay** [comment, marked as answer]
It's the positive rate as shown on the DDC dashboard. I've updated the [downloads document](https://github.com/djay/covidthailand/blob/main/downloads.md#moph_dashboad_provjson--moph_dashboard_provcsv).

You can see them labeled more clearly in the plot
![](https://github.com/djay/covidthailand/wiki/positivity_3.png)

It's unclear why ddc dashboard positive rate is consistently lower than the positive rate in the from the PCR test reports. Without knowing the process to collect each, both could be incomplete data but in different ways. If anyone knows, please let me know.

Even though DDC positive rate is lower, since the data is available per province it allows us to see trends geographically, even if the numbers are unlikely to be accurate.

![](https://github.com/djay/covidthailand/wiki/positivity_region_3.png)
